### ISSUE TITLE
pesq metric reduction parameter

### ISSUE THREAD

**Issue by wooseok-shin** [Lightning-AI/metrics#1092]
Hi, 

```
from torchmetrics.audio.pesq import PerceptualEvaluationSpeechQuality

batch_size = 4
x1 = torch.randn(batch_size, 64000)
x2 = torch.randn(batch_size, 64000) 
wb_pesq = PerceptualEvaluationSpeechQuality(reduction='none')

print(wb_pesq(x1, x2))   # result is 1.6 (scalar)
```

I want to extract pesq score for each individual instance in the batch.
For other metrics (SSIM, etc...), I found that there was a reduction parameter, but not pesq. 
Why is there no reduction='none' in pesq?

Best Regards,

**Comment by SkafteNicki** [comment]
Hi @wooseok-shin, if you want to calculate the metric per batch element you can just use the functional implementation:
```python
import torch
from torchmetrics.functional.audio import perceptual_evaluation_speech_quality
x1 = torch.randn(batch_size, 64000)
x2 = torch.randn(batch_size, 64000) 
perceptual_evaluation_speech_quality(x1, x2, fs=16000, mode='wb')
# tensor([1.2084, 2.1955, 1.4239, 1.2004], dtype=torch.float64)
```
